app/views/access_limit/edit.html.erb

```diff
@@ -41,7 +52,8 @@
             data_attributes: {
               gtm: "choose-access-limit",
               "gtm-value": t("access_limit.edit.type.all_organisations")
-            }
+            },
+            hint_text: supporting_orgs.any? && "[#{supporting_orgs.join(",")}]"
```

Comment:
If I understand right, given a primary org of GDS and supporting orgs of HMRC and DWP this would show: [HMRC, DWP]. Whereas my understanding from the [design](https://content-publisher-prototype.cloudapps.digital/access-limit) is that this should show:

[GDS]
[HMRC]
[DWP]

Reply:
You're correct. I've documented in the commit that we will support this in further work, as it depends on a [change to gov pub cub](https://github.com/alphagov/govuk_publishing_components/pull/996).

Reply:
Ah sure, might be worth including the primary one in this style too for the time being so that doesn't risk getting missed.

Reply:
Hint component renders in a `<div>` element now and Content Publisher is using the latest version of govuk_publishing_components so this could be rebased and updated to display organisations in a list element `<ul class="govuk-list">`. We may need a colour overwrite though, let me know if that's the case.

Reply:
I've updated to use the new gov pub cub release, but you're right that we need an override for the colour.

<img width="778" alt="Screen Shot 2019-07-19 at 13 43 23" src="https://user-images.githubusercontent.com/9029009/61535868-3752ce00-aa2b-11e9-9216-623eaf4bdf2e.png">

Reply:
Can you add the following attribute in `app/assets/stylesheets/utilities/_overwrites.scss`?

```
.govuk-hint {
  .govuk-list {
    color: $govuk-secondary-text-colour;
  }
}
```

I will raise it upstream, but to have it working now.